dev-docs/bidders/connatix.md

```diff
@@ -11,7 +11,7 @@ gpp_sids: tcfeu, usp
 schain_supported: false
 dchain_supported: false
 userId: none
-media_types: banner
+media_types: video, banner

```

Comment:
Can you add this meta field as well if you now support multiple formats

```
multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
```

Reply:
The `multiformat_supported` meta field is mentioned down below in the header of the file.
```
multiformat_supported: will-bid-on-one
```

Reply:
Ah damn. Sorry :smiley: